src/pages/settings/client-portal/ClientPortal.tsx

```diff
@@ -63,27 +52,11 @@ export function ClientPortal() {
     >
       {showPlanAlert && <AdvancedSettingsPlanAlert />}
 
-      <TabGroup tabs={tabs}>
-        <div>
-          <SettingsComponent />
-        </div>
-
-        <div>
-          <Authorization />
-        </div>
-
-        <div>
-          <Registration />
-        </div>
-
-        <div>
-          <Messages />
-        </div>
+      <Tabs tabs={tabs} className="mt-6" />
```

Comment:
@beganovich The reason for this change is that the `Registration` tab is missing when configuring `group_settings`. The TabGroup component requires modifications to handle cases like this. Therefore, I chose to implement tabs similar to the other tabs we have in settings pages. I accomplished this using routes. I hope you are in agreement with this approach.